kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -43,15 +44,15 @@ def _find_loop_block_device_by_file(self, nvram_install_path):
                 continue
         return None
 
-    def _make_nvram_to_block_device(self, nvram_install_path):
-        cmd = "losetup --find --show '%s'" % shlex.quote(nvram_install_path)
+    def _make_nvram_to_block_device(self):
+        cmd = "losetup --find --show '%s'" % shlex.quote(self.nvram_install_path)
         loop_dev = bash.bash_o(cmd).strip() # type: str
         if not loop_dev:
-            raise Exception("Failed to setup loop device for %s" % nvram_install_path)
+            raise Exception("Failed to setup loop device for %s" % self.nvram_install_path)
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)